infra/travis/travis_build.py

```diff
@@ -53,6 +54,7 @@ def execute_helper_command(helper_command):
 def build_fuzzers(project, sanitizer, engine):
   """Execute helper.py's build_fuzzers command on |project|. Build the fuzzers
   with |sanitizer| and |engine|."""
+  print('Building {0} with {1} and {2}'.format(project, sanitizer, engine))
```

Comment:
s/{1}/{1} sanitizer
s/{2}/{2} engine ?
just address, memory might confuse people?

Reply:
Maybe should print the command we are executing. That makes it easy for people to know what ran ?